docs/components/MessagingInputView.jsx

```diff
@@ -196,8 +208,15 @@ export default class MessagingInputView extends React.PureComponent {
             />{" "}
             Show uploaded attachments
           </label>
+          <label className={cssClass.CONFIG}>
+            <input
+              type="checkbox"
+              checked={showCheckbox}
```

Comment:
Should this be `checked={checkboxValue`?

Reply:
discussed offline that this is representing the checkbox in the config area, not the checkbox in the messagingInput